examples/cookies/README.md

```diff
@@ -0,0 +1,11 @@
+Generic Kore application that shows off a few things:
```

Comment:
The example README could use some changes as this seems copy/pasted from examples/generic.